project-base/storefront/connectors/customer/CustomerUser.ts

```diff
@@ -0,0 +1,13 @@
+import { TypeCompanyCustomerUser, TypeRegularCustomerUser } from 'graphql/types';
+import { CustomerUserType } from 'types/customer';
+
+export const getCustomerUser = (
+    customerUser: TypeCompanyCustomerUser | TypeRegularCustomerUser | undefined,
```

Comment:
I'm encountering an issue where, after editing a user, the entire user list disappears. I'm not certain if it's related to this particular line, but it seems to happen consistently after the edit operation.

Reply:
It is working now.